CMakeLists.txt

```diff
@@ -89,7 +89,7 @@ find_package(PkgConfig) # pkg_check_modules
 set(OPENRAVE_PYTHON_INSTALL_DIR)
 
 if( OPT_PYTHON )
-  include(FindPythonInterp)
+  find_package(PythonInterp 2)

```

Comment:
@cielavenir when using pybind11, i think we also allow python 3..

Reply:
as I wrote above, that is for testing. I never make pull request for that commit. Sorry for confusion.